pkg/timewindow/timewindow_test.go

```diff
@@ -40,6 +40,34 @@ func TestTimeWindows(t *testing.T) {
 			{"2019/04/18 00:00 UTC", true},
 			{"2019/04/18 23:59 UTC", true},
 		}},
+		{"mon,tue,wed,thu,fri", "9pm", "5am", "America/Los_Angeles", []testcase{
+			{"2019/04/04 22:49 PDT", true},
+			{"2019/04/05 00:49 PDT", true},
+			{"2019/04/04 00:49 PDT", true},
+			{"2019/04/05 08:59 PDT", false},
+			{"2019/04/05 9:01 PDT", false},
+			{"2019/03/31 10:00 PDT", false},
+			{"2019/04/04 12:00 PDT", false},
```

Comment:
nit: Might be worth checking if UTC still works fine with the new code for those conditions.